common/network-common/pom.xml

```diff
@@ -42,19 +42,41 @@
     </dependency>
 
     <!-- Core dependencies -->
+    <!-- Netty Begin -->
     <dependency>
       <groupId>io.netty</groupId>
       <artifactId>netty-all</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-transport-native-epoll</artifactId>
+      <classifier>linux-x86_64</classifier>
```

Comment:
Hm does this mean we actually ship both versions of the dependencies? I wonder if that causes conflicts, not sure about this

Reply:
<img width="507" alt="image" src="https://user-images.githubusercontent.com/1475305/164727082-7e7cfa87-cdda-438d-8a61-bcfbd212c100.png">

I don't think so. These jars only include the `.so` of the corresponding Linux platform.

On the other hand, they all belonged to Netty all before `Netty-4.1.68`

Reply:
I have manually `mvn test -pl core -am` on MacOS, If necessary, I can do a full test on MacOS